File: LeetCode/Easy/1678-goal-parser-interpretation/try-5.py
/*
 * Problem #1678: Goal Parser Interpretation
 * Difficulty: Easy
 * Submission: Try 5
 * status: Accepted
 * Language: python3
 * Date: 7/9/2026, 10:35:19 AM
 * Link: https://leetcode.com/problems/goal-parser-interpretation/
 */

class Solution:
    def interpret(self, command: str) -> str:
        return command.replace("()", "o").replace("(al)", "al")

File: LeetCode/Easy/485-max-consecutive-ones/try-5.c
/*
 * Problem #485: Max Consecutive Ones
 * Difficulty: Easy
 * Submission: Try 5
 * status: Accepted
 * Language: c
 * Date: 7/10/2026, 12:27:08 AM
 * Link: https://leetcode.com/problems/max-consecutive-ones/
 */

int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int n=numsSize;
    int c=1;
    int max=1;
    int hash[2]={0,0};
    for(int i=0;i<=n-1;i++){
        hash[nums[i]]++;
    }
    if(hash[1]==0){
        return 0;
    }
    for(int i=0;i<n-1;i++){
        if(nums[i]==nums[i+1] && nums[i]==1){
            c++;
        }
        else{
            c=1;
        }
        if(c>max){
            max=c;
        }
    }
    return max;

}
